web/src/editor/props.ts

```diff
@@ -5,7 +5,7 @@ export const LANGUAGE_GOLANG = 'go';
 
 export const DEMO_CODE = [
     'package main\n',
-    'import "fmt"\n',
+    'import (\n"\t\tfmt"\n)\n',
```

Comment:
One tab is enough here :)

```suggestion
    'import (',
    '\t"fmt"',
    ')',
```

Reply:
Thanks, fixed)